<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <node name="calibrated_imu_tf" pkg="tf2_ros" type="static_transform_publisher" args="0 0 0 0 0 0 1 base_link imu_link"/>
</launch>
